src/main/scala/mesosphere/marathon/api/akkahttp/EntityMarshallers.scala

```diff
@@ -132,6 +134,18 @@ object EntityMarshallers {
       }
   }.handleValidationErrors
 
+  def groupUpdateUnmarshaller(groupId: PathId)(implicit normalization: Normalization[mesosphere.marathon.raml.App]): FromEntityUnmarshaller[GroupUpdate] = {
```

Comment:
Hm, I avoided such a marshaller in the pods controller and called the validation and normalization explicitly. What is the benefit of such a marshaller?